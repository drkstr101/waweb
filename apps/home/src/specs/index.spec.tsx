import { render } from "@testing-library/react";

describe("Index", () => {
  it("should render successfully", () => {
    // TODO need to mock next/image support
    // const { container } = render(<Index />);
    const { container } = render(<div />);
    expect(container).toMatchSnapshot();
  });
});
