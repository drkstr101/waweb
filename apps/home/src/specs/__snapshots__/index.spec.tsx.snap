// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render successfully 1`] = `
<div>
  <div />
</div>
`;
